### PR TITLE
PR to incorporate maven-antrun-plugin 

### PR DESCRIPTION
**Pull request by chinmaybijwe**

### CODE REVIEW

**Comment by huntc** [comment]
Thanks, but there are conflicts preventing a merge. Can you please resolve?

**Comment by chinmaybijwe** [comment]
Done! Thanks

Chinmay

File: pom.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>strip-json-comments</artifactId>
-    <version>1.0.3-SNAPSHOT</version>
+    <version>1.0.4-SNAPSHOT</version>
```
Comment (jamesward) [inline pom.xml]: If you are going to bump the version here you also need to bump the upsteam version below.

Comment (chinmaybijwe) [inline pom.xml]: Hmm.. it seems like I should keep it at 1.0.3-SNAPSHOT.  update coming up

Comment (jamesward) [inline pom.xml]: If this is an update to `1.0.2` then the version should be `1.0.2-1-SNAPSHOT`.

**Comment by jamesward** [comment]
Note that the version still isn't correct for an update to `1.0.2`.

**Comment by chinmaybijwe** [comment]
The library version (upstream) is still at 1.0.2. I made the webjar incorrectly in the beginning. So the snapshot version should be one higher right (or am I completely getting this wrong)?

**Comment by huntc** [comment]
By convention we attempt to align the pom version with the version associated with the actual project. As @jamesward states, appending a `-1` means that we're releasing `1.0.2` again, but slightly revised.

**Comment by jamesward** [comment]
The upstream version is still `1.0.2` and this is an update to the WebJar distribution of `1.0.2` so our versioning scheme is to append a `-1` to the version.  So the version will be `1.0.2-1` but the way we do that with Maven is to set the version to `1.0.2-1-SNAPSHOT` and then Maven does the actual release.  Sorry this is confusing Maven-isms and WebJar-isms all rolled together into a confusing mess.

**Comment by chinmaybijwe** [comment]
Got it. Thanks. Update incoming.

Chinmay

**Comment by chinmaybijwe** [comment]
@jamesward: Haha! Oh well, now that I understand, hopefully it should be a lot smoother next time for me. Thanks :)

**Comment by jamesward** [comment]
Cool.  It is really my fault for not documenting this versioning scheme anywhere.  Sorry for the hassle.  Usually it is just me doing these kinds of updates so I haven't tried to educate others on this yet.

**Comment by jamesward** [comment]
This will be released shortly.

**Comment by chinmaybijwe** [comment]
:+1:

**Pull request merged**